tracer/src/Datadog.Trace/IAST/VulnerabilityType.cs

```diff
@@ -5,8 +5,14 @@
 
 namespace Datadog.Trace.Iast;
 
-internal static class VulnerabilityType
+/// <summary>
+/// Type of an aspect function
+/// </summary>
+public enum VulnerabilityType
```

Comment:
For performance reasons, using strings instead of enums was suggested:
https://github.com/DataDog/dd-trace-dotnet/pull/3225#discussion_r989174808
https://github.com/DataDog/dd-trace-dotnet/pull/3225#discussion_r989167583

Reply:
Aspect definition is easier this way (and the marshalling to native)